evaluate_test.go

```diff
@@ -100,8 +100,8 @@ var evaluateTests map[string]expressionTest = map[string]expressionTest{
 			{expression: "part in String", result: false},
 			{expression: "port not in String", result: false},
 			{expression: "part not in String", result: true},
-			{expression: "unexported == `unexported`", result: false, err: "Selector \"unexported\" is not valid"},
-			{expression: "Hidden == false", result: false, err: "Selector \"Hidden\" is not valid"},
+			{expression: "unexported == `unexported`", result: false, err: "Selector [\"unexported\"] is not valid"},
```

Comment:
This is an example of the undesirable error message changes that my other comments were talking about.

Reply:
Are they undesirable?